source/index.html.md

```diff
@@ -7505,7 +7506,6 @@ Special testing endpoints that only exist in the sandbox environment.
 ```shell
 curl --request POST \
   --url https://api.sandbox.split.cash/simulate/incoming_payid_payment \
-  --header 'accept: application/json' \
```

Comment:
This happened after removing the response body schema. I tested it, and this header is not necessary.